File: frontend/app/src/main/java/com/example/campuslink_android/data/dto/ItemResponseDto.java
package com.example.campuslink_android.data.dto

data class ItemResponseDto(
        val itemId: Int,
        val ownerId: Int,
        val title: String,
        val description: String?,
        val price: Double,
        val category: String,
        val images: List<String>? = emptyList()
)

File: frontend/app/src/main/java/com/example/campuslink_android/data/dto/ItemResponseDto.kt
package com.example.campuslink_android.data.dto

data class ItemResponseDto(
    val id: Int,
    val title: String,
    val description: String?,
    val price: Double,
    val category: String,
    val ownerId: Int,
    val images: List<String> = emptyList(),
    val createdAt: String
)
